http3/frames.go

```diff
@@ -184,3 +186,35 @@ func (f *settingsFrame) Append(b []byte) []byte {
 	}
 	return b
 }
+
+type goawayFrame struct {
+	ID quic.StreamID
+}
+
+func parseGoawayFrame(r io.Reader, l uint64) (*goawayFrame, error) {
+	if l > 8*(1<<10) {
```

Comment:
This doesn’t seem right.

Reply:
Just copied the length check from the setting frame parsing. I guess it should compare to the maximal length of a quic varint 😕 ?